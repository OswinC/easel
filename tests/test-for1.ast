
int i;
for (i = 7 ; i < 10 ; i = i + 1) {
print(i);
}
print(100);

