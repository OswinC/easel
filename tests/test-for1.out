7
8
9
100
